doc/usermanual/overview/overview.xml

```diff
@@ -85,8 +85,9 @@
     </para></listitem>
 
     <listitem><para>
-      The powerful export system supports Google+ webalbum, flickr upload, disk storage, 1:1
-      copy, email attachments and can generate a simple html-based web gallery. darktable allows
+      The powerful export system supports local disk storage, popular web albums (flickr,
+      Google Photos, Facebook webalbum, Piwigo), LaTeX book template,
```

Comment:
I don't think we want to specify Flickk, Google Photos and Facebook... Those are not working anymore and we will probably remove them.